chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -6,59 +6,97 @@
 'use strict';
 
 import {Launcher} from '../chrome-launcher';
-
+import {spy, stub} from 'sinon';
 import * as assert from 'assert';
 
 const log = require('lighthouse-logger');
 const fsMock = {
   openSync: () => {},
-  closeSync: () => {}
+  closeSync: () => {},
+  writeFileSync: () => {}
 };
 
 describe('Launcher', () => {
-  it('accepts and uses a custom path', async () => {
+  beforeEach(() => {
```

Comment:
i just tried and even without the logger set to error level, it doesn't log anything during tests.
so this might not be necessary?